test/unittests/Microsoft.ServiceFabric.Services.Tests/ServicePartitionClientTests.cs

```diff
@@ -29,6 +29,11 @@ public class ServicePartitionClientTests
 
         private static readonly TimeSpan DefaultRetryDelay = TimeSpan.FromMilliseconds(50);
 
+        // Due to bugs in BIOS or Hardware Abstraction Layer, Stopwatch sometimes reports slightly smaller value, as
+        // documented here: https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch?view=net-8.0
+        // We need to account for that possibility.
+        private static readonly long StopwatchPrecisionMs = 10;
```

Comment:
I think you wanted to increase the StopwatchPrecisionMs because a recent test run failed with the 10ms delta.